sandbox/lua/decoders/iis.lua

```diff
@@ -0,0 +1,127 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[[hekad]
+
+Parses the iis logs based on the Microsoft iis log formats. This decoder is tested for iis verions 7 and 8. 
+
+
+Config:
+
+iis_version_7 = true
+     Default configuration asssumes iis log format for version 8. 
+     For version 7 and similar formats, set the decoder config variable  iis_version_7 to true
+
+*Example Heka Configuration*
+
+.. code-block:: ini
+
+
+share_dir="C:\\heka-agent\\heka\\share\\heka"
+base_dir = "C:\\var\\cache\\hekad"
+
+[IISLogs]
+type = "LogstreamerInput"
+log_directory = "F:\\Web_Logs"
+file_match = '(?P<dir>\w+)(?P<s>\S+)u_ex(?P<Index>\d+)\.log'
+differentiator = ["dir"]
+priority = ["Index"]
+decoder = "IISDecoder"
+
+[IISDecoder]
+type = "SandboxDecoder"
+script_type = "lua"
+filename = "lua_decoders\\iis.lua"
+
+[IISDecoder.config]
+payload_keep = true
+iis_version_7 = true
+tz = "UTC"
+
+*Example Heka Message*
```

Comment:
Heading is here for an example message of what this decoder would generate, but the example itself is missing.